docs/docs-content/integrations/community_packs.md

```diff
@@ -31,15 +26,26 @@ maintain and update community packs. Therefore, updates to these packs depend on
 
 :::
 
+### Supported Environments
+
+Palette comes with a default community pack registry hosted by Spectro Cloud that is available to all multi-tenant SaaS
+deployments. For different deployment models, the registry must be added manually. Refer to the table below to check if
```

Comment:
instead of saying it is available to all deployments, it should say available to all tenants.